packages/ui-components/src/components/Form/Form.stories.js

```diff
@@ -14,8 +14,9 @@ import { Select } from "../Select/index.js"
 import { SelectOption } from "../SelectOption/index.js"
 import { Switch } from "../Switch/index.js"
 import { Textarea } from "../Textarea/index.js"
-import { RadioGroup } from "../RadioGroup/index.js"
-import { Radio } from "../Radio/index.js"
+// Temporarily disabled until TS migration is done
```

Comment:
Please do not comment code, delete it.

Reply:
And lets write this to a ticket that related to a form component as acceptance criteria.

Reply:
Added here: https://github.com/cloudoperators/juno/issues/228